src/pages/Dashboard/Settings/WidgetSettings/WidgetSettings.jsx

```diff
@@ -303,7 +318,9 @@ const WidgetSettings = () => {
   }, [initialEntitiesOrganization]);
 
   useEffect(() => {
-    const urlCopy = new URL('https://s3.ca-central-1.amazonaws.com/staging.cms-widget.footlight.io/index.html');
+    const urlCopy = new URL(widgetUrl);
+    const urlCopyMobile = new URL(widgetUrl);
```

Comment:
Similarly here

Reply:
there are 2 widgets embedded in that page and the URL used in mobile sized widget does not respond to height parameter(the other one responds to the change in height). That is the reason 2 URLs are used here.